[2025-04-03T04:39:40.733+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-03T04:39:40.783+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schedule.move_cloud_produtos_bancarios scheduled__2025-03-23T08:00:00+00:00 [queued]>
[2025-04-03T04:39:40.833+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schedule.move_cloud_produtos_bancarios scheduled__2025-03-23T08:00:00+00:00 [queued]>
[2025-04-03T04:39:40.837+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-03T04:39:41.539+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): move_cloud_produtos_bancarios> on 2025-03-23 08:00:00+00:00
[2025-04-03T04:39:41.574+0000] {standard_task_runner.py:72} INFO - Started process 411 to run task
[2025-04-03T04:39:41.598+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'schedule', 'move_cloud_produtos_bancarios', 'scheduled__2025-03-23T08:00:00+00:00', '--job-id', '1956', '--raw', '--subdir', 'DAGS_FOLDER/dag_master.py', '--cfg-path', '/tmp/tmp4agd37v6']
[2025-04-03T04:39:41.617+0000] {standard_task_runner.py:105} INFO - Job 1956: Subtask move_cloud_produtos_bancarios
[2025-04-03T04:39:41.880+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-03T04:39:42.242+0000] {task_command.py:467} INFO - Running <TaskInstance: schedule.move_cloud_produtos_bancarios scheduled__2025-03-23T08:00:00+00:00 [running]> on host dca2c4cc2d1b
[2025-04-03T04:39:42.245+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-03T04:39:42.559+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='schedule' AIRFLOW_CTX_TASK_ID='move_cloud_produtos_bancarios' AIRFLOW_CTX_EXECUTION_DATE='2025-03-23T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-23T08:00:00+00:00'
[2025-04-03T04:39:42.570+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-03T04:39:42.600+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-03T04:39:42.604+0000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'docker exec linux_python /bin/bash -c \'/workspace/scripts/run_move_file.sh /path/to/move_cloud/produtos_bancarios/produtos_bancarios.csv table_ingestion_files/produtos/20250403/produtos_1743655180.csv && echo "Script Executado com Sucesso!"\'']
[2025-04-03T04:39:42.629+0000] {subprocess.py:99} INFO - Output:
[2025-04-03T04:39:44.397+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,385 - INFO - Iniciando o upload do arquivo '/path/to/move_cloud/produtos_bancarios/produtos_bancarios.csv' para o blob 'table_ingestion_files/produtos/20250403/produtos_1743655180.csv' no container 'filedataimport'.
[2025-04-03T04:39:44.418+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,385 - INFO - Blob URL gerada: https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/produtos/20250403/produtos_1743655180.csv?sp=cw&st=2024-10-20T02:42:12Z&se=2026-10-20T10:42:12Z&spr=https&sv=2022-11-02&sr=c&sig=MaoRwZWwtKEx5vDafwkbeHkx1Rir6bWRYy0FB3OOs58%3D
[2025-04-03T04:39:44.422+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,413 - INFO - Enviando o arquivo '/path/to/move_cloud/produtos_bancarios/produtos_bancarios.csv' para o Azure Blob Storage.
[2025-04-03T04:39:44.435+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,426 - INFO - Request URL: 'https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/produtos/20250403/produtos_1743655180.csv?sp=REDACTED&st=REDACTED&se=REDACTED&spr=REDACTED&sv=REDACTED&sr=REDACTED&sig=REDACTED'
[2025-04-03T04:39:44.439+0000] {subprocess.py:106} INFO - Request method: 'PUT'
[2025-04-03T04:39:44.441+0000] {subprocess.py:106} INFO - Request headers:
[2025-04-03T04:39:44.444+0000] {subprocess.py:106} INFO -     'Content-Length': '8060'
[2025-04-03T04:39:44.446+0000] {subprocess.py:106} INFO -     'x-ms-blob-type': 'REDACTED'
[2025-04-03T04:39:44.451+0000] {subprocess.py:106} INFO -     'If-None-Match': '*'
[2025-04-03T04:39:44.453+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-04-03T04:39:44.458+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/octet-stream'
[2025-04-03T04:39:44.465+0000] {subprocess.py:106} INFO -     'Accept': 'application/xml'
[2025-04-03T04:39:44.467+0000] {subprocess.py:106} INFO -     'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.12.9 (Linux-6.10.14-linuxkit-aarch64-with-glibc2.36)'
[2025-04-03T04:39:44.468+0000] {subprocess.py:106} INFO -     'x-ms-date': 'REDACTED'
[2025-04-03T04:39:44.472+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': 'aa4a136a-1045-11f0-8597-0242ac120004'
[2025-04-03T04:39:44.473+0000] {subprocess.py:106} INFO - A body is sent with the request
[2025-04-03T04:39:44.963+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,959 - INFO - Response status: 409
[2025-04-03T04:39:44.966+0000] {subprocess.py:106} INFO - Response headers:
[2025-04-03T04:39:44.966+0000] {subprocess.py:106} INFO -     'Content-Length': '220'
[2025-04-03T04:39:44.966+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/xml'
[2025-04-03T04:39:44.966+0000] {subprocess.py:106} INFO -     'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
[2025-04-03T04:39:44.966+0000] {subprocess.py:106} INFO -     'x-ms-request-id': '41fcff4b-201e-0055-0a52-a4fe05000000'
[2025-04-03T04:39:44.967+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': 'aa4a136a-1045-11f0-8597-0242ac120004'
[2025-04-03T04:39:44.967+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-04-03T04:39:44.967+0000] {subprocess.py:106} INFO -     'x-ms-error-code': 'BlobAlreadyExists'
[2025-04-03T04:39:44.967+0000] {subprocess.py:106} INFO -     'Date': 'Thu, 03 Apr 2025 04:39:44 GMT'
[2025-04-03T04:39:44.984+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,981 - ERROR - Erro ao enviar o arquivo '/path/to/move_cloud/produtos_bancarios/produtos_bancarios.csv': The specified blob already exists.
[2025-04-03T04:39:44.986+0000] {subprocess.py:106} INFO - RequestId:41fcff4b-201e-0055-0a52-a4fe05000000
[2025-04-03T04:39:44.990+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:44.9125745Z
[2025-04-03T04:39:44.994+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-04-03T04:39:44.994+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-04-03T04:39:44.996+0000] {subprocess.py:106} INFO - RequestId:41fcff4b-201e-0055-0a52-a4fe05000000
[2025-04-03T04:39:44.999+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:44.9125745Z</Message></Error>
[2025-04-03T04:39:45.003+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,981 - ERROR - Falha no processo de upload: The specified blob already exists.
[2025-04-03T04:39:45.006+0000] {subprocess.py:106} INFO - RequestId:41fcff4b-201e-0055-0a52-a4fe05000000
[2025-04-03T04:39:45.008+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:44.9125745Z
[2025-04-03T04:39:45.009+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-04-03T04:39:45.012+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-04-03T04:39:45.015+0000] {subprocess.py:106} INFO - RequestId:41fcff4b-201e-0055-0a52-a4fe05000000
[2025-04-03T04:39:45.016+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:44.9125745Z</Message></Error>
[2025-04-03T04:39:45.190+0000] {subprocess.py:106} INFO - Script Executado com Sucesso!
[2025-04-03T04:39:45.238+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-03T04:39:45.541+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-03T04:39:45.543+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=schedule, task_id=move_cloud_produtos_bancarios, run_id=scheduled__2025-03-23T08:00:00+00:00, execution_date=20250323T080000, start_date=20250403T043940, end_date=20250403T043945
[2025-04-03T04:39:45.714+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-03T04:39:45.976+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-03T04:39:45.988+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
